modules/ROOT/pages/apikit-asyncapi-module-reference.adoc

```diff
@@ -0,0 +1,200 @@
+  <classifier>mule-plugin</classifier>
+</dependency>
+----
+[calloutlist]
+.. Check https://docs.mulesoft.com/release-notes/apikit/apikit-release-notes[APIkit release notes] to get the *LATEST_VERSION*.
+
+== Namespace and Schema
+
+----
+xmlns:apikit-asyncapi="http://www.mulesoft.org/schema/mule/apikit-asyncapi" 
+xsi:schemaLocation="http://www.mulesoft.org/schema/mule/apikit-asyncapi http://www.mulesoft.org/schema/mule/apikit-asyncapi/current/mule-apikit-asyncapi.xsd">
+----
+
+== Configurations
+
+=== Config
+
+Default configuration
+
+==== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Name | String | Name for this configuration. Connectors reference the configuration with this name | None | *Yes*
+| Api Definition | String |  API definition | None | *Yes*
+| Kafka Configs | Array of <<KafkaConfig>> | List of Kafka configurations | No |  
+| Anypoint MQ Configs | Array of <<AnypointMqConfig>> | List of AMQ configurations | No |  
+| Name | String | Identifier of this element used to reference it in other components | None | *Yes*
+|===
+
+=== Supported Message Brokers
+
+[%header%autowidth.spread]
+|===
+| Feature | Description | Supported Protocols | Configuration Details 
+| Anypoint MQ Message Broker | Supports asynchronous messaging with Anypoint MQ. | anypointmq | Anypoint MQ configurations are specified in the server object of the AsyncAPI specification. Supports both publish and subscribe operations. 
+| Kafka Message Broker | Supports asynchronous messaging with Apache Kafka. | kafka, kafka-secure | Kafka configurations are specified in the server object of the AsyncAPI specification. Supports both publish and subscribe operations. 
+|===
+
+==== Associated Operations
+* <<Publish>>  
+
+==== Associated Sources
+* <<MessageListener>>  
+
+== Operations
+
+[[Publish]]
+== Publish
+`<apikit-asyncapi:publish>`
+
+=== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use | None | *Yes* 
+| Content | Binary | None |  #[payload] | No  
+| Output MIME Type | String | MIME type of the payload that this operation outputs | None | No 
+| Output Encoding | String | Encoding of the payload that this operation outputs | None | No 
+| Config Ref | ConfigurationProvider | Name of the configuration used to execute this component | None | *Yes* 
+| Channel Name | String | Channel name | None | *Yes* 
+| Server Name | String | Server name | None | *Yes* 
+| Protocol Specific Parameters a| One of:
+
+* <<KafkaPublishParameters>> 
+
+| Protocol-specific parameters | None | No  
+| Target Variable | String | Name of a variable on which the operation's output is placed | None | No 
+| Target Value | String | Expression that is evaluated against the operation's output and the effect of that expression is stored in the target variable | #[payload] | No 
+|===
+
```

Comment:
The table row seems different from the one i shared last https://salesforce-internal.slack.com/archives/C06TECER1NV/p1716809401621239?thread_ts=1715081113.359159&cid=C06TECER1NV.  It had the following rows - Configuration, Content, Config Ref, Channel Name, Server Name, Target Variable, Target Value, Error Mappings.